atd-vzd/schema/fatalities.sql

```diff
@@ -0,0 +1,128 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 14.6 (Debian 14.6-1.pgdg110+1)
+-- Dumped by pg_dump version 14.7 (Ubuntu 14.7-0ubuntu0.22.04.1)
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- Name: fatalities; Type: TABLE; Schema: public; Owner: visionzero
+--
+
+CREATE TABLE public.fatalities (
+    id integer NOT NULL,
+    crash_id integer,
+    victim_name text,
+    ytd_fatal_crash integer,
+    ytd_fatality integer,
+    person_id integer,
+    primaryperson_id integer,
+    is_deleted boolean DEFAULT false NOT NULL,
+    CONSTRAINT either_primaryperson_or_person CHECK ((((person_id IS NULL) AND (primaryperson_id IS NOT NULL)) OR ((person_id IS NOT NULL) AND (primaryperson_id IS NULL))))
+);
+
+
+ALTER TABLE public.fatalities OWNER TO visionzero;
+
+--
+-- Name: CONSTRAINT either_primaryperson_or_person ON fatalities; Type: COMMENT; Schema: public; Owner: visionzero
+--
+
+COMMENT ON CONSTRAINT either_primaryperson_or_person ON public.fatalities IS 'Each fatality must have either a primary person or person ID but not both';
+
+
+--
+-- Name: fatalities_id_seq; Type: SEQUENCE; Schema: public; Owner: visionzero
+--
+
+CREATE SEQUENCE public.fatalities_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
```

Comment:
I don't think this is going to be a problem, but one thing that is a gotcha that I've run into before is if you somehow have this table around with records in it, and then reset its sequence to dole out IDs that end up causing inserts to fail because of a primary key or unique constraint on the ID field.

Here, where you're creating the table just above, I think you can assume that this sequence is created at the same time and will start with 1.

Reply:
huh interesting, yeah i just used the hasura console to make this table and used the default -> nextval selection for the `id` and this is how hasura automatically writes the sql for that i guess. i cant really think of how the sequence would get reset though?